selftests/job_api/test_features.py

```diff
@@ -308,6 +316,21 @@ def test_check_output_file(self):
 
     suites.append(TestSuite.from_config(config_check_output_file))
 
+    # ========================================================================
+    # Print features covered in this test
+    # ========================================================================
+    if args.features:
+        features = []
+        for suite in suites:
+            for variants in suite.config['run.dict_variants']:
+                features.append(variants['namespace'])
+
+        unique_features = sorted(set(features))
+        print('Total features covered: %i' %len(unique_features))
```

Comment:
As a small tweak, I'd show this as:

```
Features covered (21):
job.output.loglevel
...
sysinfo.collect.enabled
```

Reply:
That is much better. I have force pushed the change.